packages/bitcore-node/src/modules/firocoin/api/contract.ts

```diff
@@ -62,29 +65,71 @@ router.get('/:contractAddress', async (req, res) => {
 router.get('/:contractAddress/code', async (req, res) => {
```

Comment:
Could we return 404 in the case of not found contract?

Reply:
roger kub